### PR TITLE
Arabic shakl Added

### PR DESCRIPTION
**Pull request by chadidi**
Updated by : Abdellah Chadidi => https://github.com/chadidi

### CODE REVIEW

File: src/Carbon/Lang/ar.php [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-    'h' => '{0}ساعة|{1}ساعة|{2}ساعتين|[3,10]:count ساعات|[11,Inf]:count ساعة',
-    'minute' => '{0}دقيقة|{1}دقيقة|{2}دقيقتين|[3,10]:count دقائق|[11,Inf]:count دقيقة',
-    'min' => '{0}دقيقة|{1}دقيقة|{2}دقيقتين|[3,10]:count دقائق|[11,Inf]:count دقيقة',
-    'second' => '{0}ثانية|{1}ثانية|{2}ثانيتين|[3,10]:count ثوان|[11,Inf]:count ثانية',
-    's' => '{0}ثانية|{1}ثانية|{2}ثانيتين|[3,10]:count ثوان|[11,Inf]:count ثانية',
-    'ago' => 'منذ :time',
-    'from_now' => 'من الآن :time',
-    'after' => 'بعد :time',
-    'before' => 'قبل :time',
+    'year'  => '[0,1] سَنَة|{2} سَنَتَيْن|[3,10]:count سَنَوَات|[11,Inf]:count سَنَة',
```
Comment (chadidi) [inline src/Carbon/Lang/ar.php]: i also changed `{0}سنة|{1}سنة` with this `[0,1] سَنَة` for less code

**Comment by lucasmichot** [comment]
Thx a lot @chadidi - Can you also add tests ?

**Comment by chadidi** [comment]
@lucasmichot I don't know why but this translation failed in styleci test . and i don't know a lot about testing yet ! so ... just merge it .

**Comment by Ovsyanka** [comment]
Seems like you changed EOL characters. You should use LF here as i know.

**Comment by chadidi** [comment]
@Ovsyanka I copied this file from sublime text and past it to Github so why Github didn't change the EOL to LF if they are using this EOL version ! . and i am using windows 10 how can i fix it? do you have any suggetions ? **_sorry for my english_**

**Comment by Ovsyanka** [comment]
I never did this throught GitHub site, so ineed some toime to investigate that problem.

But as i understand you have your local repo and can push from it not using GitHub site. Am I right? You can put in sublime preferences:

```
"show_encoding": true,
"show_line_endings": true,
```

and then select Unix EOL for that file and save it: http://prntscr.com/eifptw

You can contact with me throught IRC on Freenode: https://webchat.freenode.net/, just write `/query Ovsyanka hi` after connect. I will help you if you still have troubles.

**Comment by Ovsyanka** [comment]
Oh, i am sorry. My mistake =) You didnt change EOL, you just change all this lines. But there still the broblem about styleci. I will try to figure out what is going on.

**Comment by Ovsyanka** [comment]
I realised what is the problem. Styleci, actually showing you the formatting changes you should make. You should remove auto-alignment of your code.

**Comment by GawainLynch** [comment]
​Owing to the apparently "unmaintained" status of this project (last official update was Februrary 2017), a small group have forked this project, and merged this pull request.
​​
You can safely update to the new project by changing `nesbot/carbon` to `carbondate/carbon` in your project's `composer.json` file.

The new home for the project is https://github.com/CarbonDate/Carbon and we welcome continuing engagement from the open source community.

Please also note that Carbon owes its very existence to Brian Nesbitt's work, and for that, we are eternally grateful.

**Comment by kylekatarnls** [comment]
Hi, sorry for the delay but can you pick in this file wrong sentences that your PR would fix:
https://github.com/kylekatarnls/Carbon/blob/a7223e05b203acaf0372649794f78292d896da10/tests/Localization/ArTest.php

Thanks.

**Comment by kylekatarnls** [comment]
Hi, after investigation, I know understand a bit better what the shakl is.

I understand that users can use it or not. And as Carbon use "simple" vocabulary, most arabic-speaking person won't need shakl to read them.

So if I perfectly understand it would be breaking changes for some existing applications because if they have sentences without shakl and with date in it, when they will upgrade Carbon, the shakl will appear on the Carbon words but not the words around.

Suppose this sentence exists in a webpage:
```إغلاق الخدمة في 13 دقيقة، على عجل```
(The service close in 13 minutes, hurry)
And saying "in 13 minutes" is given by Carbon, adding the shakl will produce that (if I do not mistake):
```إغلاق الخدمة في 13 دَقِيقَة، على عجل```

It will add them on "minute" in the middle of a sentence with no shakl. Does it will not feel strange?

If you agree that, I suggest to create a dedicated locale **ar_shakl.php**

**Comment by kylekatarnls** [comment]
Hi, I moved your code in a dedicated file (keeping your authorhip): #1115

It means shakl will be available with `Carbon::setLocale('ar_shakl');`

**Pull request closed without merging**